spec/factories/projects.rb

```diff
@@ -7,12 +7,27 @@
     initialize_with { Tmuxinator::Project.new(file) }
   end
 
+  factory :project_with_force_attach, :class => Tmuxinator::Project do
+    transient do
+      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
+    end
+
+    initialize_with { Tmuxinator::Project.new(file, {:force_attach => true}) }
+  end
+
+  factory :project_with_force_detach, :class => Tmuxinator::Project do
+    transient do
+      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
+    end
+    initialize_with { Tmuxinator::Project.new(file, {:force_detach => true}) }
+  end
+
   factory :project_with_custom_name, :class => Tmuxinator::Project do
     transient do
       file { YAML.load(File.read("#{File.expand_path("spec/fixtures/sample.yml")}")) }
     end
 
-    initialize_with { Tmuxinator::Project.new(file, "custom") }
+    initialize_with { Tmuxinator::Project.new(file, {:custom_name => "custom"}) }
```

Comment:
Line is too long. [81/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.